src/includes/class-alma-wc-refund.php

```diff
@@ -64,11 +64,15 @@ public function __construct() {
 	 * @return void
 	 */
 	public function admin_init() {
+		if ( isset( $_GET['post'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification
+			$maybe_wc_order = wc_get_order( intval( $_GET['post'] ) ); // phpcs:ignore WordPress.Security.NonceVerification
+			if ( $maybe_wc_order instanceof WC_order && $this->helper->is_paid_with_alma( $maybe_wc_order ) ) {
+				add_action( 'admin_notices', array( $this, 'admin_notices' ), 10 );
+				add_action( 'woocommerce_order_item_add_action_buttons', array( $this, 'woocommerce_order_item_add_action_buttons' ), 10 );
```

Comment:
Because for `admin_notices` & `woocommerce_order_item_add_action_buttons` we are in the context of displaying the BO order page, and for 

- woocommerce_order_status_changed
- woocommerce_new_order_note_data
- woocommerce_order_fully_refunded
- woocommerce_order_partially_refunded

we are during the process after the user clicked on the submit button to validate the order form. At this point there is no global `$post` or $`order` that I can use. All these 4 hooks receive the $`order_id` as a parameter, so the test has to be done inside the callback functions.